projects/storefrontstyles/scss/components/product/search/_searchbox.scss

```
@@ -296,6 +296,17 @@
         z-index: 20;
         padding-top: 25px;
       }
+
+      @include forFeature('a11yActiveAndFocusedSearchBoxComponent') {
+        &:focus-within,
+        &:active {
+          outline-style: solid;
+          outline-color: var(--cx-color-visual-focus);
+          outline-width: var(--cx-visual-focus-width, 2px);
+          outline-offset: 4px;
+          transition: none;
```

We should already have a mixin for the focus outline, can it be used here?
```suggestion
 @include visible-focus();
```

---

I've just updated it.